src/main/java/edu/ksu/canvas/attendance/form/InputValidator.java

```diff
@@ -0,0 +1,48 @@
+package edu.ksu.canvas.attendance.form;
+
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+@Component
+public class InputValidator implements Validator {
+
+    @Override
+    public boolean supports(Class<?> clazz) {
+        return CourseConfigurationForm.class.isAssignableFrom(clazz);
+    }
+
+    @Override
+    public void validate(Object target, Errors errors) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "validate" is 22 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)